layout/_partial/index.ejs

```diff
@@ -32,8 +32,14 @@
         <div class="postShorten-wrap">
             <% if (thumbnailImageUrl !== null && thumbnailImagePosition === "top") { %>
                 <a href="<%- url_for(post.path) %>">
-                    <div class="postShorten-thumbnailimg">
-                        <img alt="" itemprop="image" src="<%= thumbnailImageUrl %>"/>
+                    <div class="postShorten-thumbnailimg" itemprop="image" itemscope itemtype="https://schema.org/ImageObject">
+                        <img alt="" src="<%= thumbnailImageUrl %>"/>
+                        <meta itemprop="url" content="<%= thumbnailImageUrl %>">
+                        <% // width and height are required ImageObject, values hardcode should have way to populate from
+                           // css variables 
+                        %>
+                        <meta itemprop="width" content="140">
```

Comment:
140 px was derived from CSS files. This shall be the same for display and metadata, I did not figure out a way to grab information from css and throw it here dynamically during theme build. This would be ideal to avoid mismatches in future. However, it might not harm.